web/blueprints/finance/__init__.py

```diff
@@ -810,7 +810,7 @@ def transactions_unconfirmed():
         unconfirmed_transactions_table=UnconfirmedTransactionsTable(
             data_url=url_for(".transactions_unconfirmed_json"))
     )
-
+"""TODO: Add post mapping which forges the Payload send by JS and accepts the selected"""
```

Comment:
`TODO` comment, please fix or remove (+track via issue if relevant)
```suggestion
```

Nota bene: Use comments for this (`# TODO`) instead of a multiline string literal (`"""TODO"""`), as there is no need to create a python string object.

However, for dostrings, it is correct to use a string instead of a comment, because strings which are specified directly below a function `f()` are assigned to its `f.__doc__` attribute, which makes `help` and autogeneration of API documentation (e.g. via `sphinx`) work.